### PR TITLE
Require react-native to access NativeModules.

### PR DESCRIPTION
**Pull request by jedlau**
Addresses #7 .

### CODE REVIEW

**Comment by adamjmcgrath** [comment]
Hi, thanks for your contribution. You can fix the tests by updating the simpleAuthClient stub to:

``` js
simpleAuthClient = proxyquire('../lib/simpleauthclient', {
  'react-native': {
    NativeModules: {
      SimpleAuthWrapper: {
        configure: configureSpy,
        authorize: authorizeSpy
      }
    },
    '@noCallThru': true
  }
});
```

in https://github.com/adamjmcgrath/react-native-simple-auth/blob/master/test/simpleauthclient.test.js#L14

**Comment by jedlau** [comment]
Sorry about that. I'll push those changes, but locally, I get the following error:

```
  1) SimpleAuthClient "before all" hook:
     Error: Cannot find module 'ActivityIndicatorIOS'
      at Object.<anonymous> (lib/simpleauthclient.js:3:25)
      at Context.<anonymous> (test/simpleauthclient.test.js:14:24)
```

File: test/simpleauthclient.test.js [hunk 1]
```diff
@@ -12,12 +12,14 @@ describe('SimpleAuthClient', () => {
...
-          authorize: authorizeSpy
-        },
-        '@noCallThru': true
+      'react-native': {
+        NativeModules: {
+          SimpleAuthWrapper: {
+            configure: configureSpy,
+            authorize: authorizeSpy
+          },
+          '@noCallThru': true
```
Comment (adamjmcgrath) [inline test/simpleauthclient.test.js]: This should be a property of 'react-native'

Comment (jedlau) [inline test/simpleauthclient.test.js]: Thanks! Tests pass now. Just pushed.

**Pull request merged**